01/components/App.js

```diff
@@ -8,16 +8,12 @@ class App extends React.Component {
     };
 
     componentDidMount() {
-        this.intervalId = setInterval(() => {
+        setInterval(() => {
             const date = new Date();
             this.setState({ date });
         }, 1000);
     }
 
-    componentWillUnmunt() {
-        clearInterval(this.intervalId);
-    }
-
```

Comment:
Należy czyścić komponenty - to raczej powinno zostać, tylko jest źle w metodzie jest błąd w nazwie :P